deeds-dapp-webapp/src/main/webapp/vue-app/dapp/js/authorizationCodeService.js

```diff
@@ -37,8 +37,8 @@ export function sendEmailConfirmation(email) {
   });
 }
 
-export function isCodeValid(code) {
-  return fetch(`${window.parentAppLocation}/api/authorization`, {
+export function isCodeValid(code, email) {
+  return fetch(`${window.parentAppLocation}/api/authorization?email=${email}`, {
```

Comment:
Never add email, password, username in clear text inside a GET HTTP Method or any other HTTP URL Method.

```suggestion
  return fetch(`${window.parentAppLocation}/api/authorization`, {
```